src/utils/helpers.js

```diff
@@ -103,6 +100,9 @@ export const formatThousand = (num) => {
   return num.toFixed(2);
 };
 
+/* eslint-disable-next-line no-console */
+export const debugLog = (...logs) => network && console.log(...logs);
```

Comment:
Add separate flag ENABLE_DEBUG_LOG. To enable/disable it manually

Reply:
I add env variable NODE_ENV
NODE_ENV = 'development' => dev branch
NODE_ENV = 'production' => master branch

we will not be able to see the production logs in this case. 
if you watn enable\disable logs locally u must change "network" to "! network" here  ****(...logs) => **network** && console.log(...logs)****

Reply:
> must change "network" to "! network" here

Reply:
Debug log feature should be separate from dev/prod selection

Reply:
okay